src_analysis/change_utils.py

```diff
@@ -58,6 +60,30 @@ def get_changed_fuzzers_for_ci(changed_files: List[str] = None) -> List[str]:
     return get_changed_fuzzers(changed_files)
 
 
+def get_changed_fuzzers_since_last_experiment():
```

Comment:
Maybe I should put this function in automatic_run_experiment (its only caller) if I don't want src_analysis to depend on database/

Reply:
Did this.